brion/plugin/spikeReportASCII.h

```diff
@@ -45,6 +44,12 @@ class SpikeReportASCII : public SpikeReportPlugin
 protected:
     Spikes _spikes;
     Spikes::iterator _lastReadPosition;
+
+    // Returns true if parsing succeeded
+    using ParseFunc = std::function< bool( const std::string&, Spike& ) >;
+
+    static Spikes parse( const Strings& files, const ParseFunc& parse );
+    static Spikes parse( const std::string& file, const ParseFunc& parse );
```

Comment:
The same approach  should be followed for file writing. In the current implementation SpikeReportNEST and SpikeReportBluron duplicate all the code but one line.